apache/superset
# feat(sqllab): Add timeout on fetching query results

### SUMMARY
This commit adds the timeout on fetching query results.
Additionally, a 'Retry without timeout' option is provided so that when a timeout issue occurs, users can attempt to proceed by ignoring the error.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://github.com/user-attachments/assets/af5dfc12-4072-484a-94d8-57b3258b3163

### TESTING INSTRUCTIONS
SET SQLLAB_QUERY_RESULT_TIMEOUT with a small timeout
Run a query async
Check the error message and then click "Retry" to get the result without timeout errors

### ADDITIONAL INFORMATION
